cli/assets/templates/go/src/main.go

```diff
@@ -1,26 +1,30 @@
 package main
 
-var smiley = [8]byte {
-    0b11000011,
-    0b10000001,
-    0b00100100,
-    0b00100100,
-    0b00000000,
-    0b00100100,
-    0b10011001,
-    0b11000011,
-};
+import "cart/src/w4"
+
+var smiley = [8]byte{
+	0b11000011,
+	0b10000001,
+	0b00100100,
+	0b00100100,
+	0b00000000,
+	0b00100100,
+	0b10011001,
+	0b11000011,
+}
+
+func main() {}
 
 //go:export update
-func update () {
-    *DRAW_COLORS = 2;
-    text("Hello from Go!", 10, 10);
+func update() {
+	*w4.DRAW_COLORS = 2
+	w4.Text("Hello from Go!", 10, 10)
 
-    var gamepad = *GAMEPAD1;
-    if (gamepad & BUTTON_1 != 0) {
-        *DRAW_COLORS = 4;
-    }
+	var gamepad = *w4.GAMEPAD1
+	if gamepad&w4.BUTTON_1 != 0 {
+		*w4.DRAW_COLORS = 4
+	}
 
-    blit(&smiley[0], 76, 76, 8, 8, BLIT_1BPP);
-    text("Press X to blink", 16, 90);
+	w4.Blit(&smiley[0], 76, 76, 8, 8, w4.BLIT_1BPP)
+	w4.Text("Press X to blink", 16, 90)
```

Comment:
4 spaces indentation plz :nerd_face:

Reply:
No, Go uses Tabs for identation and spaces for aligning variables etc.. Using the "gofmt" tool will enforce this. For that reason, the Spaces vs Tabs war is pretty much non-existent in Go.

Reply:
Ok, makes sense.